pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -369,6 +369,17 @@ message LinuxContainerConfig {
     optional Capability capabilities = 2;
     // Optional SELinux context to be applied.
     optional SELinuxOption selinux_options = 3;
+    // User contains the user for the container process.
+    optional LinuxUser user = 4;
```

Comment:
ha, now that I think about this I had the same thought reading my code (too much Golang..)